src/app/mat-select-search/mat-select-search.component.html

```diff
@@ -1,8 +1,10 @@
 
 <input matInput class="mat-select-search-input mat-select-search-hidden"/>
 
-<div class="mat-select-search-inner"
-     [ngClass]="{'mat-select-search-inner-multiple': matSelect.multiple}">
+<div
+      #innerSelectSearch
+      class="mat-select-search-inner mat-typography mat-datepicker-content mat-header-row"
```

Comment:
why is the `mat-datepicker-content` and `mat-header-row` class needed?

Reply:
Well as i said the 3rd one war relatively hard for me to solve. The problem is the User can make their own custom themes. As far as i know @angular/material2 does not expose those properties so u can use them, because they see it as "source code". I worked around it by using classes that does only affect the background-color, color and border-bottom-color.
The "mat-datepicker-content" inherits the background-color and color property and the mat-header-row sets the border-bottom-color. Maybe you have a better idea?

Reply:
ah, ok, great workaround though ;-)
one would have to introduce a custom template mixin scss file, which a user would need to include in their global styles. the mixin could then use the material theme variable. would you like to implement this, too? if not, I think your existing solution is also fine.

Reply:
Yeah i thought about that solution too but i think it wouldn't be very user friendly. I would rather leave it as it is. The main goal in the end is that angular/material2 includes their own solution anyway, so... I guess we are good for now :)

Reply:
ok, great. so I will test it tomorrow and then release a new version. are you fine with that?

Reply:
Sure!